content/applications/productivity/documents.rst

```diff
@@ -246,16 +242,22 @@ Upload <documents/upload>` files to it.
 
 .. _documents/upload:
 
-Upload by email
-^^^^^^^^^^^^^^^
+Files upload
+^^^^^^^^^^^^
+
+Toggle the :guilabel:`Allow upload` switch to allow users to upload files to your workspace.
+
+Tick the :guilabel:`Upload by Email` checkbox to enable users to upload documents sent by email in
+the workspace.
+
+.. note::
 
-Select the :guilabel:`Download and Upload` option to enable users to upload their files to your
-workspace using an :guilabel:`Email Alias`. To create the email alias, enter its name in the
-:guilabel:`Email Alias` field. The :doc:`domain name <../general/email_communication/email_servers>`
-should be set by default, but you can modify it by clicking it.
+   Your **email server** must be configured to be able to create an **email alias**. If it isn't
+   configured yet, click :guilabel:`Choose or Configure Email Servers` and configure the
+   :doc:`mail server <../../applications/general/email_communication/email_servers>`.
```

Comment:
I think you can remove the blank line one line 254 as it's not necessary.

And sorry for insisting, but I really think the information gets lost between the two note boxes (+ the tip below the second one).
![image](https://github.com/user-attachments/assets/e7fd7e42-0226-482c-930d-410128f34699)

I know you mentioned that people might be lost if they haven't configured the email server yet, but I believe it should be fine. There's a good chance they will have already configured the server before reaching this feature anyway. Could we move the info to the second note box, as another list item, please?